tsconfig.json

```diff
@@ -1,5 +1,11 @@
 {
   "compilerOptions": {
-    "module": "commonjs"
-  }
+    "module": "commonjs",
+    "target": "ESNext",
+    "strictNullChecks": true
+  },
+  "typeRoots": [
+    "./src/interfaces",
```

Comment:
This should be types now. 😉

Reply:
Maybe best to delete this b/c it does not appear to affect my linter.